homeassistant/components/reolink/services.py

```diff
@@ -0,0 +1,81 @@
+"""Reolink additional services."""
+
+from __future__ import annotations
+
+from reolink_aio.api import Chime
+from reolink_aio.enums import ChimeToneEnum
+from reolink_aio.exceptions import InvalidParameterError, ReolinkError
+import voluptuous as vol
+
+from homeassistant.config_entries import ConfigEntryState
+from homeassistant.const import ATTR_DEVICE_ID
+from homeassistant.core import HomeAssistant, ServiceCall
+from homeassistant.exceptions import HomeAssistantError, ServiceValidationError
+from homeassistant.helpers import device_registry as dr
+
+from .const import DOMAIN
+from .host import ReolinkHost
+from .util import get_device_uid_and_ch
+
+ATTR_RINGTONE = "ringtone"
+
+
+async def async_setup_services(hass: HomeAssistant) -> bool:
```

Comment:
This doesn't have to be a coroutine function since we don't await inside. Make it an async callback.

Reply:
Done, converted to a callback